services/ui_backend_service/requirements.txt

```diff
@@ -1,7 +1,6 @@
-aiohttp==3.6.2
-aiohttp-swagger==1.0.15
-boto3==1.15.10
```

Comment:
I think the reason for pinning the boto3 version had something to do with the dependencies introduced with the cache client, will check what this was about.